src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets

```diff
@@ -107,8 +104,55 @@ Copyright (c) .NET Foundation. All rights reserved.
       GeneratePublishBlazorBootJson;
     </GenerateStaticWebAssetsPublishManifestDependsOn>
 
+    <ResolveCompressedFilesDependsOn>
+      $(ResolveCompressedFilesDependsOn);
+      _ResolveBlazorWasmOutputs;
+      _AddBlazorWasmStaticWebAssets;
+    </ResolveCompressedFilesDependsOn>
+
+    <ResolveCompressedFilesForPublishDependsOn>
+      $(ResolveCompressedFilesForPublishDependsOn);
+      ProcessPublishFilesForBlazor;
+      ComputeBlazorExtensions;
+      _AddPublishBlazorBootJsonToStaticWebAssets;
+      _CollectAssetsToCompress;
+    </ResolveCompressedFilesForPublishDependsOn>
+
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <_BlazorBuildGZipCompressDirectory>$(IntermediateOutputPath)build-gz\</_BlazorBuildGZipCompressDirectory>
+    <_BlazorPublishCompressDirectory>$(IntermediateOutputPath)compress\</_BlazorPublishCompressDirectory>
```

Comment:
This should just be defined by the compression targets. There is no need for consumers to make a choice on this.

Ideally, all compressed assets end up in `obj\fx\config\compressed\(publish?)` in a flat list of files where the name is a "hash" of the relative path of the asset + the .br or .gz extension.

Move [this](https://github.com/dotnet/sdk/blob/a821467d6ef6f4d3f9c3de8c468041d8c8d5b3b0/src/WasmSdk/Tasks/FileHasher.cs) class around and use it to compute the name (instead of what we currently use, which I think is base64 + some character replacements).

Reply:
Sounds good. I've changed it so the consumer no longer needs to specify anything. When this PR is closer to its final state, I'll change the output directories to match what you wrote so the baseline updates happen at the end.